frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/common/constants.ts

```diff
@@ -9,6 +9,7 @@ export const SPLIT_TEXTAREA_TRANSFORM = {
 
 export const SPLIT_TEXTAREA_VALIDATION = {
   validate: (opts: string) => {
+    if (!opts) return 'Please provide at least 1 option.'
     const optsArr = SPLIT_TEXTAREA_TRANSFORM.output(opts)
```

Comment:
Let's use the existing mechanism that decides what constitutes an option instead. With it, whitespaces string will also be handled.
```suggestion
    const optsArr = SPLIT_TEXTAREA_TRANSFORM.output(opts)
    if (!optsArr.length) return 'Please provide at least 1 option.'
```